01_install_requirements.sh

```diff
@@ -135,7 +135,9 @@ sudo python -m pip install ansible=="${ANSIBLE_VERSION}"
 
 pushd ${METAL3_DEV_ENV_PATH}
 ansible-galaxy install -r vm-setup/requirements.yml
-ansible-galaxy collection install --upgrade ansible.netcommon ansible.posix ansible.utils community.general
+# Let's temporarily pin these collections to the latest compatible with ansible-2.15
+#ansible-galaxy collection install --upgrade ansible.netcommon ansible.posix ansible.utils community.general
+ansible-galaxy collection install 'ansible.netcommon<=7.2.0' ansible.posix 'ansible.utils<=5.1.2' community.general
```

Comment:
Did they actually broke the compatibility in 5.1.3? If not, I think we need to allow for patch releases and do `<7.3.0` and `<5.2.0` instead. WDYT?

Reply:
We could do ansible.utils<6.0.0  and ansible.netcommon<8.0.0 as those are the ones bringing the breaking changes. I just went with the latest released (and tested) versions to be on the safe side.

Reply:
My worry is about missing future bug fixes. Would you consider a follow-up once we merge this and unblock the CI?